packages/services/storage/src/schema-change-meta.ts

```diff
@@ -203,6 +203,8 @@ export function schemaChangeFromSerializableChange(
       return buildUnionMemberAddedMessageFromMeta(change);
     case 'REGISTRY_SERVICE_URL_CHANGED':
       return buildRegistryServiceURLFromMeta(change);
+    default:
+      throw new Error(`Unsupported change type: ${change.type}`);
```

Comment:
Instead of throwing, we should handle this kind of change, in order to support it in Hive.